Automattic/jetpack
# E2E tests: don't send notifications for canceled jobs

#### Changes proposed in this Pull Request:
* If the e2e job was canceled a Slack notification is still sent, with an (expected) error `There was a problem parsing the test results file.` This fixes it by only sending a notification for jobs that are not canceled.

#### Jetpack product discussion
n/a

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* Run the tests in CI and cancel the job. Make sure the Send Slack notification step is skipped.
